mistletoe/base_renderer.py

```diff
@@ -44,6 +44,7 @@ class BaseRenderer(object):
 
     def __init__(self, *extras):
         self.render_map = {
+            'HTMLAttributes': self.render_html_attributes,
```

Comment:
These changes to the BaseRenderer aren't necessary and should be avoided imho. Better use the existing mechanisms for extensibility:
- The render_map entry is added automatically when you pass in HTMLAttribute into the HTMLRenderer constructor in the HTMLAttributesRenderer constructor, so you can safely remove that line.
- The render_html_attribute method can be moved to the HTMLAttributesRenderer class right away.

Reply:
See (cd6ff447f92ea39b5972ef08bb81cdd21a0dc1ff)